specifyweb/frontend/js_src/lib/localization/resources.ts

```diff
@@ -839,4 +839,7 @@ export const resourcesText = createDictionary({
     'ru-ru': 'Требуется текущее определение.',
     'uk-ua': 'Потрібне поточне визначення.',
   },
+  invalidSpecifyUser: {
+    'en-us': 'Invalid specify user',
```

Comment:
capitalize Specify
even better - don't hardcode table names in the UI localization - table names should be coming from schema localization

see how we do that in other places:
https://github.com/specify/specify7/blob/00f63833a3ec941907f5e2ac66fc8c3407976c45/specifyweb/frontend/js_src/lib/localization/forms.ts#L131-L140
and then search for usages of paleoRequiresGeography in the code to see how it get's called - do something similar here too if possible

Reply:
I think we can just reuse this string instead of using table names. What do you think?
https://github.com/specify/specify7/blob/b893ef792d0d4195156569e8c29690e798ddcdc1/specifyweb/frontend/js_src/lib/localization/forms.ts#L604-L611